nodes/index.adoc

```diff
@@ -17,6 +17,17 @@ Having stable and healthy nodes in a cluster is fundamental to the smooth functi
 In {product-title}, you can access, manage, and monitor a node through the `Node` object representing the node.
 Using the OpenShift CLI (`oc`) or the web console, you can perform the following operations on a node.
 
+Following are the components of nodes responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
```

Comment:
Following are the components of the node

Reply:
addressed